src/styles/styles.css

```diff
@@ -17,12 +17,12 @@
 .modalStyle p{
     margin:0%;
 }
-.modalStyle ul{
-    margin:0%;
-    padding-left: 0%;
+.modalStyleMentee .modalStyleMentor ul{
```

Comment:
Is the modalStyleMentor a child of modalStyleMentee? That's what this selector seems to imply. I think what you're intending is:
`.modalStyleMentee ul, .modalStyleMentor ul`